db/seeds.rb

```diff
@@ -47,12 +47,12 @@
 
 if Region.all.empty?
   puts 'Loading regions'
-  Rake::Task['shf:load_regions'].invoke
+  Rake::Task['shf:load_regions'].execute
```

Comment:
I live and learn: the reason #execute even worked is because the :environment dependency is not needed when the Rake tasks are run from code (duh). Thanks for catching that Ashley. I reverted to #invoke, and solved the problem by calling #reenable on the tasks in the stepdefinition.